docs/source/quickstart-pt-2.mdx

```diff
@@ -0,0 +1,172 @@
+
+Replace the contents of `index.js` with the following:
+
+```js:title=index.js
+const { ApolloServer } = require('apollo-server');
+const { ApolloGateway } = require('@apollo/gateway');
+
+const gateway = new ApolloGateway({});
+
+const server = new ApolloServer({
+  gateway,
+  // Subscriptions are not currently supported in Apollo Federation
+  subscriptions: false
+});
+
+server.listen().then(({ url }) => {
+  console.log(`🚀 Gateway ready at ${url}`);
+}).catch(err => {console.error(err)});
+```
+
+So now, how do we connect and authenticate our gateway with Apollo Studio? With a **graph API key** that we set as the value of an **environment variable**.
+
+> **API keys are secret credentials.** Never share them outside your organization or commit them to version control. Delete and replace API keys that you believe are compromised.
+
+1. Obtain a graph API key for your Studio graph by <a href="https://www.apollographql.com/docs/studio/api-keys/#graph-api-keys" target="_blank">following these steps</a>.  Copy its value.
+
+2. Create a new file named `.env` in your Node.js project folder.
+
+3. Paste the following into `.env` and replace `YOUR_API_KEY` with your graph API key:
+
+    ```shell
+    APOLLO_KEY=YOUR_API_KEY
+    ```
+
+4. Add the `dotenv` Node.js library to your project:
+
+    ```shell
+    npm install dotenv
+    ```
+
+    This library takes care of reading environment variable values from your `.env` file.
+
+5. Add the following line to the _very top_ of `index.js`:
+
+  ```js
+  require('dotenv').config()
+  ```
+
+6. Start up your gateway with `node index.js`. You'll see some extra output now:
+
+    ```
+    No graph variant provided. Defaulting to `current`.
+    🚀 Gateway ready at http://localhost:4000/
+    Apollo usage reporting starting! See your graph at https://studio.apollographql.com/graph/docs-example-graph/?variant=current
+    ```
+
+> By providing an API key to your gateway, you also automatically enable **federated trace reporting** to Apollo Studio, enabling you to view helpful performance metrics. [Learn more about federated traces.](./metrics/)
+
+Now, open up GraphQL Playground in your browser again at `localhost:4000`. Sure enough, your supergraph schema's types and fields are all available, and you can execute test queries as before. But now, all of this information is coming from Apollo Studio.
+
+## Moving forward
+
+Nice work! We have two **subgraphs** registering their schemas with Apollo, and we have a **federated gateway** that then fetches the composed **supergraph schema**.
+
+If we were now to register changes to one of our subgraph schemas, our running gateway would automatically fetch the corresponding changes to the supergraph schema. However, because we're currently using Apollo-hosted services that we can't modify, it doesn't make sense to register changes to our subgraph schemas.
```

Comment:
this last sentence was a bit confusing, suggest it's stronger to omit and end with this:
```suggestion
If we were now to register changes to one of our subgraph schemas, our running gateway would automatically fetch the corresponding changes to the supergraph schema.
```